src/kontrol/options.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+    from pathlib import Path
+
+    from pyk.utils import BugReport
+
+    from .foundry import Foundry
+
+
+@dataclass
+class GlobalOptions:
```

Comment:
I'm wondering if the class name should reflect the fact that these settings hold throughout each `prove` run, e.g., `ProveOptions`? I'm not sure, so please feel free to ignore that.

Reply:
Renamed to `ProveOptions`.